react-app/src/library/services/vendor-implementations/sennheiser/sennheiser.ts

```diff
@@ -190,12 +190,19 @@ export default class SennheiserService extends VendorImplementation {
   }
 
   endCall (conversationId: string): Promise<void> {
+    this._sendMessage({
+      Event: SennheiserEvents.Resume,
+      EventType: SennheiserEventTypes.Request
+    });
+    this._sendMessage({
+      Event: SennheiserEvents.UnmuteFromApp,
+      EventType: SennheiserEventTypes.Request,
+    });
```

Comment:
This pretty much just sets the headset device state to vanilla (i.e. mute and hold are set to false) so that future calls start off with a clean slate.

Reply:
Is this going to cause issues in multicall scenarios? Does it make sense to make use of the `hasOtherActiveCalls` property being provided to `endCall`?